apps/evm/src/pages/Account/Settings/index.tsx

```diff
@@ -1,15 +1,13 @@
 import { Card, Icon, Toggle } from 'components';
-import { CHAIN_METADATA } from 'constants/chainMetadata';
+import { useGetChainMetadata } from 'hooks/useGetChainMetadata';
 import { useIsFeatureEnabled } from 'hooks/useIsFeatureEnabled';
 import { useUserChainSettings } from 'hooks/useUserChainSettings';
 import { useTranslation } from 'libs/translations';
-import { useChainId } from 'libs/wallet';
 import Section from '../Section';
 
 export const Settings: React.FC = () => {
   const { t } = useTranslation();
-  const { chainId } = useChainId();
-  const { name: chainName } = CHAIN_METADATA[chainId];
+  const chainMetadata = useGetChainMetadata();
```

Comment:
Shouldn't this be destructured like before?

Reply:
Sure